apps/mymedicare_cb/views.py

```diff
@@ -105,14 +105,10 @@ def callback(request):
     response = requests.get(url, cert=certs, verify=False)
 
     if 'entry' in response.json():
-        identifiers = response.json()['entry'][0]['resource']['identifier']
-        fhir_id = ""
-        for i in identifiers:
-            if i['system'] == 'http://bluebutton.cms.hhs.gov/identifier#bene_id':
-                fhir_id = i['value']
-            if fhir_id:
-                cx.fhir_id = fhir_id
-                cx.save()
+        fhir_id = response.json()['entry'][0]['resource']['id']
```

Comment:
Your lookup seems correct. Ran the following command locally and your "`response.json()['entry'][0]['resource']['id']`" path matches the result I got:

    $ curl --silent --insecure --cert-type pem --cert files/client-test-keypair.pem "https://fhir.backend.bluebutton.hhsdevcloud.us/baseDstu3/Patient?identifier=http%3A%2F%2Fbluebutton.cms.hhs.gov%2Fidentifier%23hicnHash%7C57cb89fea33c344d69cf0deed7153c3810cc8a870ab82009fc83fd9a1351b4a7&_format=application%2Fjson%2Bfhir"

Reply:
Sample output from that: https://pastebin.com/qDwsibJQ